docs/installation/linux.md

```diff
@@ -79,7 +79,7 @@ sudo installer -pkg powershell-6.0.0-alpha.9.pkg -target /
 Paths
 =====
 
-* `$PSHOME` is `/opt/microsoft/powershell/6.0.0-alpha.9/`
+* `$PSHOME` is `/usr/local/microsoft/powershell/6.0.0-alpha.9/`
```

Comment:
`$PSHOME` is `/opt/microsoft/powershell/<version>/` on Ubuntu. So here we probably need to specify on which OS, maybe like `$PSHOME is ... on Ubuntu; ... on OS X; ... on CentOS`?